src/main/java/ca/corefacility/bioinformatics/irida/model/irida/IridaSequenceFile.java

```diff
@@ -25,6 +25,13 @@
 	 * @return the physical location of the file.
 	 */
 	public Path getFile();
+	
+	/**
+	 * Returns whether or not the file is gzipped.
+	 * 
+	 * @return True if gzipped, false otherwise.
+	 */
+	public boolean isGzipped();
```

Comment:
This class (Along with all the Irida* model interfaces) really aught to be deprecated and removed.  Was the Galaxy stuff using `IridaSequenceFile` already?  If so we can just remove it in a later merge.

Reply:
Added an issue for removing them #405

Reply:
Yes, the Galaxy code was using `IridaSequenceFile`. Thanks for making another issue.